csharp/src/Ice/ObjectAdapter.cs

```diff
@@ -40,6 +44,23 @@ public sealed class ObjectAdapter : IDisposable, IAsyncDisposable
         /// determines this protocol.</summary>
         public Protocol Protocol { get; }
 
+        /// <summary>Returns the endpoints listed in a direct proxy created by this object adapter.</summary>
```

Comment:
Maybe add `<see cref="SetPublishedEndpoints"/> and <see cref="SetPublishedEndpointsAsync"/>` in the summary here

Reply:
Done.